stable/kong/ci/loadbalancer-values.yaml

```diff
@@ -0,0 +1,44 @@
+# CI test for LoadBalancer admin/proxy types
+
```

Comment:
Can we introduce a `default-values.yaml` to run the chart with the default values?

Reply:
Oops - I thought the test script would still run a version with `values.yaml`.

I just added a symlink from `ci/default-values.yaml` -> `values.yaml`, in the hopes that will work so there won't be two copies of the same file to maintain.

If that doesn't work... I guess duplication is the answer.